src/BackTube.js

```diff
@@ -112,7 +112,7 @@ export default class BackTube {
     }
   }
 
-  onYouTubeIFrameAPIReady() {
+  onYouTubeIFrameAPIReady: Function = () => {
```

Comment:
This is a little tedious but seems like its currently required for exported classes. Issue 1757 in flow.
However, when used in react, this doesn't seem to be required.